src/packer.json

```diff
@@ -6,7 +6,7 @@
           "delete_on_termination": true,
           "device_name": "/dev/sda1",
           "encrypted": true,
-          "volume_size": 20,
+          "volume_size": 30,
```

Comment:
I see the commit about this, but I would like you to add a mention for this in the PR description if you don't mind.

Reply:
Take a look now - I added a sentence or two.

Reply:
Perfect, thanks. I just wanted a mention for posterity 👍